lte/gateway/c/oai/lib/3gpp/3gpp_38.413.h

```diff
@@ -191,7 +191,7 @@ typedef struct PDU_Session_resource_setup_request_s {
 // Direction: NG-RAN node → AMF
 typedef struct PDU_Session_resource_setup_response_s {
   Ngap_Message_Type_t Ngap_Message_Type;
-  amf_ue_ngap_id_t
+  amf_ue_ngap_id_ty
```

Comment:
why is this changed? how does this compile?

Reply:
Fixed